core/modules/date/date.admin.inc

```diff
@@ -400,22 +400,22 @@ function _date_field_widget_settings_form($field, $instance) {
       'within' => t('Within'),
       'none' => t('None'),
     );
-    $description = t("The location of date part labels, like 'Year', 'Month', or 'Day' . 'Above' displays the label as titles above each date part. 'Within' inserts the label as the first option in the select list and in blank textfields. 'None' doesn't visually label any of the date parts.");
   }
   else {
     $options = array(
-      'none' => t('Hidden'),
       'above' => t('Above'),
+      'none' => t('Hidden'),
     );
-    $description = t('The location of date part labels, like "Year", "Month", or "Day" . "Above" displays the label as titles above each date part. "Hidden" doesn\'t visually label any of the date parts.');
   }
   $form['advanced']['label_position'] = array(
     '#type' => 'radios',
     '#options' => $options,
     '#default_value' => $settings['label_position'],
     '#title' => t('Position of date part labels'),
-    '#description' => $description,
   );
+  $form['advanced']['label_position']['above']['#description'] = t("Displays the label as titles above each date part (like 'Year', 'Month', or 'Day').");
```

Comment:
This sounds a bit weird to me, having 'label' (singular) and 'titles' (plural)... How about this?
```suggestion
  $form['advanced']['label_position']['above']['#description'] = t("Displays the label as individual titles above each date part (like 'Year', 'Month', or 'Day').");
```